site/profile/manifests/core/cmms.pp

```diff
@@ -6,7 +6,7 @@
 ) {
   include profile::core::letsencrypt
 
-  $fqdn  = $facts[fqdn]
+  $fqdn  = fact('networking.fqdn')
```

Comment:
same here, and for the other occurrences. I wouldn't change it.

Reply:
:hankey: